src/client/components/household-information/FinancialDisclosureSection.jsx

```diff
@@ -33,17 +33,24 @@ class FinancialDisclosureSection extends React.Component {
       content = (<fieldset>
         <legend>Financial Disclosure</legend>
         <div className="input-section">
-          <p>In the next 4 sections you will be asked to provide your financial information from the most recent tax year. This
-          information is used to determine your eligibility for health care. Your most recent
-          tax filing will have all the information you need in it.</p>
+          <p>You will now be asked to provide your financial information from the
+          most recent tax year. We ask for this information for 2 reasons:</p>
+
+          <ol>
```

Comment:
This looks great and makes perfect sense to me. Lets go ahead and link the words "qualifying eligibility factor" with the hover list.